x/bank/keeper/keeper_test.go

```diff
@@ -2474,6 +2476,187 @@ func (suite *KeeperTestSuite) TestGetAllSendEnabledEntries() {
 	})
 }
 
+func (suite *KeeperTestSuite) TestGetAllBalances() {
```

Comment:
in these 2 new tests, can we add a check that the returned balances are sorted? (i.e. copy the return value, sort, compare against original return)

Reply:
All the expected values are created using `NewCoins` which sorts them. So if an actual result isn't sorted, it won't match the expected and the test will fail. But I added some checks anyway.